src/src/gas/station/hillel/PetrolStation.java

```diff
@@ -0,0 +1,35 @@
+package gas.station.hillel;
+
+
+import java.util.concurrent.Semaphore;
+
+public class PetrolStation {
+    private int amount;
+    private final Semaphore semaphore;
+
+    public PetrolStation(int amount) {
+        this.amount = amount;
+        this.semaphore = new Semaphore(3, true);
+    }
+
+    public void doRefuel(int requestedAmount) throws InterruptedException {
```

Comment:
remove `throws InterruptedException` and use try catch inside this method

Reply:
1. Find a way to remove `throws InterruptedException`, do some refactoring, if necessary.
2. Wrap your code with only 1 try-catch block.